Rahul-Das-Sarma/Test-Arc
# Pull Request for Component Workspace

### Pull Request for **Component Workspace**

**Description:**
-  What is the purpose of this PR?  
  _This PR introduces the creation of a new "component" workspace feature and changes the location of the public folder._

**Changes Made:**
- Key modifications:
  - **New Feature:** Created a workspace named "component" for editing purposes.
  - **Folder Structure Update:** Moved the `public` folder from inside the `src` folder to the root of the directory.

**Type of Changes:**

**Testing Instructions:**
- How to test these updates:
  1. _Step 1: Navigate to the root of the project and verify the new location of the `public` folder._
  2. _Step 2: Test the functionality of the "component" workspace to ensure it allows editing as expected._

**Screenshots :**
![git-1](https://github.com/user-attachments/assets/ea1426cc-3cd3-4cf7-8276-dc948debd04a)


**Checklist:**
- [x] Code adheres to the project's style guidelines.
- [x] Unit tests are updated to reflect the changes (if applicable).
- [x] Documentation is updated to include the new workspace and folder structure changes.
- [x] The PR is reviewed and ready for merge.
